src/components/EditCampaign.js

```diff
@@ -98,6 +103,7 @@ class EditCampaign extends React.Component {
     }
 
     async updateCampaign(){
+        console.log('update campaign called');
```

Comment:
nit: remove or comment out this log statement

Reply:
removed